src/test/java/pixelitor/guides/GuidesTest.java

```diff
@@ -75,4 +79,38 @@ void addRelativeGrid() {
         assertThat(guides.getHorizontals()).containsOnly(0.25, 0.5, 0.75);
         assertThat(guides.getVerticals()).containsOnly(0.25, 0.5, 0.75);
     }
+
+    /**
+     * Newly added test
```

Comment:
I see that this is a newly added test even without the unnecessary comment. Other than that, these tests could be useful. I assume that they are running correctly.